docs/content/guides/upgrade-and-migration/release-notes.md

```diff
@@ -24,6 +24,51 @@ See the full history of changes made to Handsontable in each major, minor, and p
 
 [[toc]]
 
+## 14.2.0
+
+Released on March 6, 2024
+
+For more information on this release, see:
+- [Blog post (14.1.0)](https://handsontable.com/blog/handsontable-14-2-0-PLACEHOLDER)
+- [Documentation (14.2)](https://handsontable.com/docs/14.2)
+
+### Added
+
+- Added a new Handsontable hook, [`beforeBeginEditing`](@/api/hooks.md#beforebeginediting), to conditionally control when to allow cell editing. [#10699](https://github.com/handsontable/handsontable/pull/10699)
+- Added the ability to prevent viewport scrolling by using two existing Handsontable hooks: [`beforeViewportScrollVertically`](@/api/hooks.md#beforeviewportscrollvertically) and [`beforeViewportScrollHorizontally`](@/api/hooks.md#beforeviewportscrollhorizontally). [#10724](https://github.com/handsontable/handsontable/pull/10724)
+- Added undo-and-redo support for column moving. [#10746](https://github.com/handsontable/handsontable/pull/10746)
+- Added a new [`Filters`](@/api/filters.md) plugin hook: [`modifyFiltersMultiSelectValue`](@/api/hooks.md#modifyfiltersmultiselectvalue). Now, filtered values lists show formatted numbers instead of raw data. [#10756](https://github.com/handsontable/handsontable/pull/10756)
```

Comment:
need to adjust this to my suggested change also, essentially it should be more clear its only numeric